packages/core/src/types.ts

```diff
@@ -51,6 +51,7 @@ export type NormalizedOutputOptions = {
   tsconfig?: Tsconfig;
   packageJson?: PackageJson;
   headers: boolean;
+  filterTag?: string[];
```

Comment:
What do you think about having this.

```ts
filters?: {
  tags?: string[]
}
```

Like that we can add other filters like on the operation id.

What do you think about adding it in the `input` section and not the `output` one?

Reply:
I agree with changing it to have a filters object, that would make more sense. I will be updating this asap.

About the `input`  section I'm not sure. I think it is more of a question how we see it:

1. We only "read in" the relevant filtered elements.
2. We only "output" the relevant filtered elements.

The first one would imply adding it in the `input` section, the second one the `output` section.
The main reason I chose to add it to the output section is since the function that  I changed in the `api.ts` file only receives `NormalizedOutputOptions` as parameter. I could of course add the `NormalizedInputOptions` as parameter to get the desired change, or do you think there would be a smarter way to go about this?

Reply:
I would do the filter [here](https://github.com/anymaniax/orval/blob/master/packages/core/src/generators/verbs-options.ts#L202) and so we don't even read the operation. And in that case, you can also pass the `NormalizedInputOptions ` as a parameter.

Reply:
I have fixed it as you proposed :)

Reply:
Really nice works! Thanks for the pull request